master/internal/resourcemanagers/resource_pool.go

```diff
@@ -381,36 +381,136 @@ func (rp *ResourcePool) receiveAgentMsg(ctx *actor.Context) error {
 	return nil
 }
 
-func (rp *ResourcePool) moveJob(msg job.MoveJob) error {
-	// REMOVEME
-	fmt.Println(rp.config.PoolName, "moveJob: ", msg)
-	rp.queuePositions[job.TailAnchor] = initalizeQueuePosition(time.Now())
-	newPos, rebalance, err := computeNewJobPos(msg, rp.queuePositions)
-	if err != nil {
-		return err
+func (rp *ResourcePool) moveJob(
+	ctx *actor.Context, jobID model.JobID, anchorID model.JobID, aheadOf bool,
+) error {
+	if rp.config.Scheduler.GetType() != priorityScheduling {
+		return nil
```

Comment:
should this return something like `ErrUnsupported` or something, rather than just doing nothing?